pkgs/development/python-modules/python-neutronclient/default.nix

```diff
@@ -0,0 +1,98 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  # Build and Runtime
+  pbr,
+  cliff,
+  debtcollector,
+  iso8601,
+  netaddr,
+  openstacksdk,
+  osc-lib,
+  oslo-i18n,
+  oslo-log,
+  oslo-serialization,
+  oslo-utils,
+  os-client-config,
+  keystoneauth1,
+  python-keystoneclient,
+  requests,
+  hacking,
+  # Tests
+  bandit,
+  coverage,
+  fixtures,
+  flake8-import-order,
+  openstackclient,
+  oslotest,
+  osprofiler,
+  subunit,
+  requests-mock,
+  stestr,
+  testtools,
+  testscenarios,
+  tempest,
+}:
+
+buildPythonPackage rec {
+  pname = "python-neutronclient";
+  version = "11.3.1";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-U82ZI/Q6OwdypA41YfdGVa3IA4+QJhqz3gW2IR0S7cs=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    pbr
```

Comment:
https://opendev.org/openstack/python-neutronclient/src/commit/a10390c62e3405ae604206b60bca4703fda8da2c/setup.py#L19
`pbr` should go to `build-system`.